Formula/git-credential-libsecret.rb

```diff
@@ -0,0 +1,37 @@
+class GitCredentialLibsecret < Formula
+  desc "Git helper for accessing credentials via libsecret"
+  homepage "https://git-scm.com"
+  # NOTE: Please keep these values in sync with git.rb when updating.
```

Comment:
Let's add this comment to `git.rb` too. We can add it in a separate PR after this is merged to save CI time.